### ISSUE TITLE
Error while executing: am start -n "com.aykuttasil.callrecorder

### ISSUE THREAD

**Issue by vondel** [aykuttasil/CallRecorder#16]
02/13 13:31:26: Launching app
$ adb install-multiple -r -t C:\Users\voeger\AndroidStudioProjects\CallRecorder-master\app\build\intermediates\split-apk\debug\slices\slice_0.apk C:\Users\voeger\AndroidStudioProjects\CallRecorder-master\app\build\intermediates\split-apk\debug\slices\slice_2.apk C:\Users\voeger\AndroidStudioProjects\CallRecorder-master\app\build\intermediates\split-apk\debug\slices\slice_8.apk C:\Users\voeger\AndroidStudioProjects\CallRecorder-master\app\build\intermediates\split-apk\debug\slices\slice_3.apk C:\Users\voeger\AndroidStudioProjects\CallRecorder-master\app\build\intermediates\split-apk\debug\slices\slice_9.apk C:\Users\voeger\AndroidStudioProjects\CallRecorder-master\app\build\intermediates\split-apk\debug\dep\dependencies.apk C:\Users\voeger\AndroidStudioProjects\CallRecorder-master\app\build\intermediates\split-apk\debug\slices\slice_1.apk C:\Users\voeger\AndroidStudioProjects\CallRecorder-master\app\build\intermediates\split-apk\debug\slices\slice_4.apk C:\Users\voeger\AndroidStudioProjects\CallRecorder-master\app\build\intermediates\split-apk\debug\slices\slice_7.apk C:\Users\voeger\AndroidStudioProjects\CallRecorder-master\app\build\intermediates\split-apk\debug\slices\slice_6.apk C:\Users\voeger\AndroidStudioProjects\CallRecorder-master\app\build\intermediates\split-apk\debug\slices\slice_5.apk C:\Users\voeger\AndroidStudioProjects\CallRecorder-master\app\build\outputs\apk\debug\app-debug.apk 
Split APKs installed
$ adb shell am start -n "com.aykuttasil.callrecorder/com.cb.callrecorder.ContactsActivity" -a android.intent.action.MAIN -c android.intent.category.LAUNCHER
Error while executing: am start -n "com.aykuttasil.callrecorder/com.cb.callrecorder.ContactsActivity" -a android.intent.action.MAIN -c android.intent.category.LAUNCHER
Starting: Intent { act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] cmp=com.aykuttasil.callrecorder/com.cb.callrecorder.ContactsActivity }
Error type 3
Error: Activity class {com.aykuttasil.callrecorder/com.cb.callrecorder.ContactsActivity} does not exist.

Error while Launching activity

Hello !

I got the following error. Please can you help me`?

bg

**Comment by aykuttasil** [comment]
Hi @vondel ,
You must click Start Record button after launch the sample project. Then, you can see the file in CallRecorderTest folder. If you want to change folder name, you must call CallRecordObject.changeRecordDirName("DirName").

**Comment by vondel** [comment]
Hello Aykuttasil! The applilcation is running but the app doens't create a folder. So i tryed to change the path. But I cant find the syntax in you proramm wherei can change it.

**Comment by aykuttasil** [comment]
Can you share the code you're running? @vondel

**Comment by vondel** [comment]
I download your last Version and make an import in Studio 3.0. Compile on Samson Galaxy S5 Neo with 
Andriod 6.0.1

**Comment by tvhieuit** [comment]
Hello I got same issue 
```
        println(" ${callRecord!!.recordDirName}")
        println(" ${callRecord!!.recordFileName}")
        println(" ${callRecord!!.recordDirPath}")
```
```
recorder
RecordFile
/storage/emulated/0
```

But I checked in folder then i got ```Filename : RecordFile_xxx_xxx```

**Comment by Ranjeet4android** [comment]
Yes saved file name and get recordFileName both are different  So Tell me How to get file is exactly.
